sdk/servicebus/service-bus/samples-v1/typescript/src/receiveMessagesStreaming.ts

```diff
@@ -30,7 +30,7 @@ export async function main() {
   // If receiving from a Subscription, use `createSubscriptionClient` instead of `createQueueClient`
   const queueClient = sbClient.createQueueClient(queueName);
 
-  // To receive messages from sessions, use createSessionReceiver instead of createReceiver or look at
+  // To receive messages from sessions, use acceptSession instead of createReceiver or look at
```

Comment:
This is a sample for v1, should not be touched in this PR :)
```suggestion
  // To receive messages from sessions, use createSessionReceiver instead of createReceiver or look at
```